src/poetry_to_uv/convertor.py

```diff
@@ -182,7 +182,13 @@ def _write_main_dependencies(
 
         uv_dependencies = tomlkit.array()
         for dependency, version in sorted_main_dependencies.items():
-            uv_dependencies.add_line(f"{dependency} == {version}")
+            if version is None:
```

Comment:
Think it might be worth updating the callers [resolve_dependency](https://github.com/PacificGilly/poetry_to_uv/blob/001bb5884ec2d7e99328cd2d5996bfee86171704/src/poetry_to_uv/convertor.py#L62-L63) to stop adding these to the UVResolvedDependencies list, to avoid having to handle this downstream here?